test/ctia/stores/es/init_test.clj

```diff
@@ -87,6 +87,52 @@
       (is (= "2s" refresh_interval)))
     (index/delete! conn (str indexname "*"))))
 
+(deftest get-existing-indices-test
+  (index/delete! es-conn (str indexname "*"))
+  (let [successful? (atom true)
+        fake-exit (fn [] (reset! successful? false))]
+    (with-redefs [sut/system-exit-error fake-exit]
+      (let [test-fn (fn [msg
+                         input-indexname
+                         expected-successful?
+                         expected-output]
+                      (reset! successful? true)
+                      (testing msg
+                        (let [ouput (sut/get-existing-indices es-conn input-indexname)]
+                        (when expected-successful?
+                          (is (= expected-output ouput)))
+                        (is (= expected-successful? @successful?)))))
```

Comment:
This part is confusing, maybe the second `is` should be slurped in, no?

Reply:
see my comment below.